modules/vba_documents/app/services/vba_documents/slack/messenger.rb

```diff
@@ -0,0 +1,37 @@
+# frozen_string_literal: true
+
+require 'common/client/base'
+
+module VBADocuments
+  module Slack
+    class Messenger
+      ALERT_URL = Settings.vba_documents.slack.default_alert_url
+      ENVIRONMENT_EMOJIS = { production: 'rotating_light', sandbox: 'rocket', staging: 'construction',
+                             development: 'brain' }.freeze
+
+      def initialize(params)
+        @params = params
+      end
+
+      def notify!
+        Faraday.post(ALERT_URL, request_body, request_headers)
+      end
+
+      private
+
+      attr_reader :params
+
+      def notification
+        VBADocuments::Slack::DefaultNotification.new(params)
```

Comment:
We only have the `DefaultNotification` for now, but could easily add custom notifications going forward, like Appeals does.